lib/validation-schemas.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { readFileSync } = require('fs');
+const { join } = require('path');
+const { runInNewContext } = require('vm');
+const { all: decorators } = require('./decorators');
+const { Metaschema, addSchema, linkSchemas } = require('./schema');
+
+const schemasDir = join(__dirname, '..', 'schemas');
+const schemas = ['systemDomains', 'DatabaseField', 'Domain'].map(name => {
+  const path = join(schemasDir, 'metaschema', `${name}.schema`);
```

Comment:
Could we perhaps reuse `schema-fs-loader/loadSchema` here, as it looks very similar?

If the `sync` part is the issue then we can add `loadSchemaSync` or something similar. This will still reuse most of the code.

Reply:
Yes, I think we should add `loadSync` and use it here